src/utils/queries/index.mjs

```diff
@@ -197,11 +197,11 @@ createQueries.QUERIES = {
   // Fixes the references to Markdown pages into the API documentation
   markdownUrl: /^(?![+a-zA-Z]+:)([^#?]+)\.md(#.+)?$/,
   // ReGeX to match the {Type}<Type> (API type references)
-  // eslint-disable-next-line no-useless-escape
-  normalizeTypes: /(\{|<)(?! )[a-zA-Z0-9.| \[\]\\]+(?! )(\}|>)/g,
+
+  normalizeTypes: /(\{|<)(?! )[^<({})>]+(?! )(\}|>)/g,
```

Comment:
What was the issue with the current regexes? Have you added tests to ensure these new ones don't accidentally capture/match things they shouldn't match? 👀

Reply:
I'd appreciate if you could! That'd increase the confidence in future changes of the code. To be blunt, it's my fault there aren't test suites for this. So thank you if you implement some!

Reply:
Noted. I'll add test cases for _all_ of our RegExps and queries in #366.